junit-platform-commons/src/main/java/org/junit/platform/commons/util/FatalExceptions.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.commons.util;
+
+import static org.apiguardian.api.API.Status.INTERNAL;
+
+import org.apiguardian.api.API;
+
+/**
+ * Internal utilities for working with <em>fatal</em> exceptions.
```

Comment:
Instead of the term "fatal", I would suggest something along the lines of "unrecoverable" meaning an exception from which the application (test plan) cannot recover.

It's certainly debatable due to the common use of "fatal errors" in computing in general, but I have been told that terms (potentially) implying "death" or "violence" should be avoided if other terms are more suitable.

Reply:
Granted, I realize that "fatal" can also mean ["causing failure"](https://www.merriam-webster.com/dictionary/fatal). That's why I say it's debatable.

Reply:
I agree that "unrecoverable" is better and will change it accordingly.